File: app/src/main/java/com/pluu/event/RxBusProvider.java
package com.pluu.event;

/**
 * RxBusProvider from RxJava
 * Created by PLUUSYSTEM-NEW on 2016-03-21.
 */
public class RxBusProvider {
    private static final RxBus BUS = new RxBus();

    private RxBusProvider() {
        // No instances.
    }

    public static RxBus getInstance() {
        return BUS;
    }
}

File: app/src/main/java/com/pluu/event/RxBusProvider.kt
package com.pluu.event

/**
 * RxBusProvider from RxJava
 * Created by pluu on 2017-04-19.
 */
class RxBusProvider private constructor() {

    companion object {
        private val _instance = RxBus()

        fun getInstance() = _instance

    }
}
